flexmeasures/data/services/scheduling.py

```diff
@@ -115,7 +115,9 @@ def make_schedule(
     db.engine.dispose()
 
     sensor = Sensor.query.filter_by(id=sensor_id).one_or_none()
-    data_source_name = "Seita"
+    data_source_info = dict(
+        name="Seita", model="Unknown", version="1"
+    )  # will be overwritten by scheduler choice below
```

Comment:
It looks like we only overwrite the model in some cases. It would be cleaner to get the `__author__` and `__version__` from our own scheduler modules, too (otherwise, changing the versions in those modules would be ignored). And then these lines can be removed.

Reply:
Agreed. The next PR should unify this into a proto-function store, and handle getting this information in a unfied way.